code/game/objects/items/kitchen.dm

```diff
@@ -153,6 +153,11 @@
 	w_class = WEIGHT_CLASS_NORMAL
 	custom_price = 600
 
+/obj/item/kitchen/knife/hunting
+	name = "hunting knife"
+	desc = "Despite its name, it's mainly used for cutting meat from dead prey rather than actual hunting."
+	icon_state = "survivalknife"
+

```

Comment:
this seems unnecessary if you're not actually changing anything about it

Reply:
flavor is important

Reply:
dab